pandas/tests/reshape/test_pivot.py

```diff
@@ -1638,3 +1639,21 @@ def test_crosstab_tuple_name(self, names):
 
         result = pd.crosstab(s1, s2)
         tm.assert_frame_equal(result, expected)
+
+    @pytest.mark.parametrize("names, input_data, expected_data_out", [
+        (['a', 'b'], [[1, 2, 3], [1, 1, 1]], [1, 1, 1]),
+        ([('a', 'b'), 'c'], [[1, 2, 2], [1, 1, 1]], [1, 2]),
+        ([('a', 'b'), ('c', 'd')], [[1, 2, 3], [1, 2, 3]],
+         np.eye(3, dtype=int))])
+    def test_crosstab_cols_output(self, names, input_data, expected_data_out):
+        row_series = pd.Series(input_data[0], name=names[0])
+        col_series = pd.Series(input_data[1], name=names[1])
+        expected_crosstab = pd.DataFrame(
+            expected_data_out,
```

Comment:
call this expected

Reply:
Is there a naming convention to be comply to in pandas, apart from pep8? I don't see the benefit of changing from expected_crosstab to expected. What's the reason behind this required change?